### DISCUSSION TITLE
Check if the Actions runner has access to the GitHub Actions service:  FAIL

### DISCUSSION THREAD

**Discussion by owyman** [community/community#36805] [Actions]
I have tried spinning up multiple self-hosted runners and when I check the services the Actions service fails. I can't hit the api endpoint https://api.github.com/api/v3/zen. The log shows the following message: {
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
and the documentation link is very broad.
<img width="885" alt="Screen Shot 2022-10-20 at 11 04 49 AM" src="https://user-images.githubusercontent.com/39495253/196996813-42887c22-e5e7-4eca-9177-7ff679a65677.png">

**Comment by ossanna16** [comment]
Thank you for asking a good question @owyman, I'm glad you found our community 🙂
